lib/ramble/ramble/experiment_set.py

```diff
@@ -93,6 +96,29 @@ def __init__(self, workspace):
         self.set_base_var(self.batch_submit_key, workspace.batch_submit)
         self.set_base_var(self.spec_name_key, '{application_name}')
 
+    def read_site_vars(self, workspace):
+        site_vars = self.get_site_vars(workspace)
+        site_env_vars = self.get_site_env_vars(workspace)
+        site_name = 'site'  # FIXME: DRY out these magic strings
+        self._set_context(self._contexts.site,
+                          site_name,
+                          site_vars,
+                          site_env_vars)
+
+    def get_site_vars(self, workspace):
+        site_conf = ramble.config.config.get_config('config', scope='site')
```

Comment:
What happens here if you leave off `scope='site'` and just get all of the config defined variables instead?

If that works, I think it would be preferable (i.e. this could be just config file defined variables, instead of specifically variables defined in the site config scope).

Then, the config parser will handle merging them before they are passed to this code.